ebusd-2.1.x/de/encon/3c.Excellent300.csv

```diff
@@ -27,133 +27,133 @@ r,,TotalFlow,Gesamtdurchsatz,,,4050,25,,,UIR,-1000,m³,,min,,UIR,-1000,m³,,max,
-w,,CO2Sensor3LowerLimit,CO2Sensor3UntererGrenzwert,,,4080,38,,,UIR,,ppm,[min:400;max:2000;step:25;default:400]
-r,,CO2Sensor3LowerLimit,CO2Sensor3UntererGrenzwert,,,4050,38,,,UIR,,ppm,,min,,UIR,,ppm,[min:400],max,,UIR,,ppm,[max:2000],step,,UIR,,ppm,[step:25],default,,UIR,,ppm,[default:400]
-w,,CO2Sensor3UpperLimit,CO2Sensor3ObererGrenzwert,,,4080,39,,,UIR,,ppm,[min:400;max:2000;step:25;default:1200]
-r,,CO2Sensor3UpperLimit,CO2Sensor3ObererGrenzwert,,,4050,39,,,UIR,,ppm,,min,,UIR,,ppm,[min:400],max,,UIR,,ppm,[max:2000],step,,UIR,,ppm,[step:25],default,,UIR,,ppm,[default:1200]
-w,,CO2Sensor4LowerLimit,CO2Sensor4UntererGrenzwert,,,4080,3a,,,UIR,,ppm,[min:400;max:2000;step:25;default:400]
-r,,CO2Sensor4LowerLimit,CO2Sensor4UntererGrenzwert,,,4050,3a,,,UIR,,ppm,,min,,UIR,,ppm,[min:400],max,,UIR,,ppm,[max:2000],step,,UIR,,ppm,[step:25],default,,UIR,,ppm,[default:400]
-w,,CO2Sensor4UpperLimit,CO2Sensor4ObererGrenzwert,,,4080,3b,,,UIR,,ppm,[min:400;max:2000;step:25;default:1200]
-r,,CO2Sensor4UpperLimit,CO2Sensor4ObererGrenzwert,,,4050,3b,,,UIR,,ppm,,min,,UIR,,ppm,[min:400],max,,UIR,,ppm,[max:2000],step,,UIR,,ppm,[step:25],default,,UIR,,ppm,[default:1200]
-w,,CO2SensorsActivated,CO2SensorenAktiviert,,,4080,3c,,,UIR,0=no;1=yes,,[min:0;max:1;step:1;default:0]
-r,,CO2SensorsActivated,CO2SensorenAktiviert,,,4050,3c,,,UIR,0=no;1=yes,,,,,IGN:6,,,,default,,UIR,0=no;1=yes,,[default:0] - min/max/step fields of enum message omitted
+w,,EWTTempMin,Min. Temp. Erdreichwärmetauscher,,,4080,19,,,SIR,10,°C,[min:0;max:100;step:5;default:50]
+r,,EWTTempMin,Min. Temp. Erdreichwärmetauscher,,,4050,19,,,SIR,10,°C,,min,,SIR,10,°C,[min:0],max,,SIR,10,°C,[max:100],step,,SIR,10,°C,[step:5],default,,SIR,10,°C,[default:50]
+w,,EWTTempMax,Max. Temp. Erdreichwärmetauscher,,,4080,1a,,,SIR,10,°C,[min:150;max:400;step:5;default:250]
+r,,EWTTempMax,Max. Temp. Erdreichwärmetauscher,,,4050,1a,,,SIR,10,°C,,min,,SIR,10,°C,[min:150],max,,SIR,10,°C,[max:400],step,,SIR,10,°C,[step:5],default,,SIR,10,°C,[default:250]
+w,,RHTSensorPresent,Feuchtesensor vorhanden,,,4080,32,,,UIR,,,[min:0;max:1;step:1;default:0]
+r,,RHTSensorPresent,Feuchtesensor vorhanden,,,4050,32,,,UIR,,,,min,,UIR,,,[min:0],max,,UIR,,,[max:1],step,,UIR,,,[step:1],default,,UIR,,,[default:0]
+w,,RHTSensorSensitivity,Empfindlichkeit Feuchtesensor,,,4080,33,,,SIR,,,[min:-2;max:2;step:1;default:0]
+r,,RHTSensorSensitivity,Empfindlichkeit Feuchtesensor,,,4050,33,,,SIR,,,,min,,SIR,,,[min:-2],max,,SIR,,,[max:2],step,,SIR,,,[step:1],default,,SIR,,,[default:0]
+w,,BacklightLevel,Beleuchtung Display,,,4080,1d,,,UIR,,%,[min:0;max:100;step:5;default:10]
+r,,BacklightLevel,Beleuchtung Display,,,4050,1d,,,UIR,,%,,min,,UIR,,%,[min:0],max,,UIR,,%,[max:100],step,,UIR,,%,[step:5],default,,UIR,,%,[default:10]
+w,,CO2Sensor1LowerLimit,CO2-Sensor 1, unterer Grenzwert,,,4080,34,,,UIR,,ppm,[min:400;max:2000;step:25;default:400]
```

Comment:
with comma in comment, it needs to be put in double quotes

Reply:
oh, I haven't noticed this one. Thanks for noticing. I will fix those.

Reply:
@john30 Fixed! 🙏